content/04.discussion.md

```diff
@@ -1,48 +1,45 @@
 ## Discussion
 
-The CBTN released the raw genomic data for the PBTA in September 2018 without embargo to allow researchers immediate access to begin making discoveries on behalf of children with CNS tumors everywhere.
-Since the release of the raw data, the CBTN has approved over 200 data research projects [@doi:10.1016/j.neo.2022.100846] from 69 different institutions, with 60% from non-CBTN sites.
-We created OpenPBTA to define an open, real-time, reproducible analysis framework to genomically characterize pediatric brain tumors, bringing together basic and translational researchers, clinicians, and data scientists on behalf of accelerated discovery and clinical impact.
-We provide robust reusable code and data resources, paired with cloud-based availability of source and derived data resources, to the pediatric oncology community, encouraging interdisciplinary scientists to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
-To our knowledge, this initiative represents the first large-scale, collaborative, open analysis of genomic data coupled with open manuscript writing, in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date, comprising 1,074 tumors across 58 distinct histologies.
-We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and establish over 40 scalable modules to perform common downstream cancer genomics analyses, all of which have undergone rigorous scientific and analytical code review.
-We detected and showed expected patterns of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
+The CBTN released the PBTA raw genomic data in September 2018 without embargo to accelerate therapeutic translation for children with cancer.
+Since this release, the CBTN has approved over 200 data research projects [@doi:10.1016/j.neo.2022.100846] from 69 different institutions, with 60% from non-CBTN sites.
```

Comment:
```suggestion
Since this publication, the CBTN has approved over 200 data research projects [@doi:10.1016/j.neo.2022.100846] from 69 different institutions, with 60% from non-CBTN sites.
```
should we use "publication" or "manuscript" here instead?